src/Serialization/Fields.jl

```diff
@@ -283,24 +283,25 @@ end
 @register_serialization_type AbstractAlgebra.Generic.RationalFunctionField "RationalFunctionField" uses_id
 
 function save_object(s::SerializerState,
-                     RF::AbstractAlgebra.Generic.RationalFunctionField)
+                     RF::AbstractAlgebra.Generic.RationalFunctionField{<: FieldElem, <: MPolyRingElem})
   save_data_dict(s) do
     syms = symbols(RF)
     save_object(s, syms, :symbols)
   end
 end
 
+function save_object(s::SerializerState,
+                     RF::AbstractAlgebra.Generic.RationalFunctionField{<:FieldElem, <: PolyRingElem})
+  save_data_dict(s) do
+    save_object(s, symbols(RF)[1], :symbol)
```

Comment:
```suggestion
    save_object(s, only(symbols(RF)), :symbol)
```
To notice if this returns more symbols than expected